transportable-udfs-test/transportable-udfs-test-spi/src/main/java/com/linkedin/transport/test/spi/SqlFunctionCallGenerator.java

```diff
@@ -53,6 +54,8 @@ default String getFunctionCallArgumentString(Object argument, TestType argumentT
       return getBooleanArgumentString((Boolean) argument);
     } else if (argumentType instanceof StringTestType) {
       return getStringArgumentString((String) argument);
+    } else if (argumentType instanceof DoubleTestType) {
+      return getDoubleArgumentString((Double) argument);
```

Comment:
What about the other new types (Binary and Float)?

Reply:
Because we can not create Binary and Float constant in SQL text :). This is for creating StdData from SQL string, right?

Reply:
Seems there could be? https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types#LanguageManualTypes-Literals. Also, not sure if casting can be used to force some data types?

Reply:
Added type cast for others.

Binary, float, double work for Hive/Spark.

Only double works for Presto due to syntax differences.

Reply:
Thanks! For engine-specific syntax, you may check extensions of SqlFunctionCallGenerator.

Reply:
Fixed, float and binary work for Presto now (need to fix the current implementation other than just overriding SqlFunctionCallGenerator)